### ISSUE TITLE
Marquee Mode & Item Sizes

### ISSUE THREAD

**Issue by mortscode** [vadimkorr/svelte-carousel#84]
Question.

~I'm trying to create an auto-scrolling image marquee, but when I set the `autoplayDuration` to `0`, the carousel stops. Is there a way for me to have this work like a smooth scrolling marquee without the starts & stops?~

**Comment by mortscode** [comment]
Figured it out with the easing prop!

Now i'm having issues with the `ParticlesToShow` prop. Is there a way I can bypass this and let the block sizing definitions of my items determine the size of each carousel item? I'm trying to create a very basic carousel of circle images with consistent spacing between them. My hope would be that there could be 1.5 in the view width on mobile and 4.5 on desktop

**Comment by vadimkorr** [comment]
Hi @mortscode,
This is the upcoming feature (https://github.com/vadimkorr/svelte-carousel/issues/82). The idea is to pass dictionary with screen width and number of particles per page...

But I guess it can be done another way,
Create particlesToShow variable, add window resize event listener and change particlesToShow depending on screen width as you want, pass particlesToShow-variable as a prop

I gon't remember is particlesToShow reactive or not - if not you can add this workaround 

```html
{#key particlesToShow }
  <Carousel particlesToShow={particlesToShow}>
    // some code...    
  </Carousel>
{/key}
```

**Comment by vadimkorr** [comment]
TODO: add demo for Marquee Mode with easing prop

**Comment by vadimkorr** [comment]
Released 1.0.18
